packages/edit-site/src/components/layout/hooks.js

```diff
@@ -5,15 +5,9 @@ import { useEffect, useState } from '@wordpress/element';
 import { useSelect } from '@wordpress/data';
 import { store as coreStore } from '@wordpress/core-data';
 
-/**
- * Internal dependencies
- */
-import useEditedEntityRecord from '../use-edited-entity-record';
```

Comment:
I actually removed this hook entirely. I introduced it initially (my bad) but It was not a good abstraction. It was either doing too much or it was something that can be done inline.